code/common/finspace.q

```diff
@@ -41,3 +41,10 @@ notifyhdb:{[cluster;changeset]
       .aws.update_kx_cluster_databases[string[cluster];.aws.sdbs[.aws.db[.finspace.database;changeset[`id];.aws.cache["CACHE_1000";"/"]]];.aws.sdep["NO_RESTART"]]
       // TODO - Also need to figure out the ideal logic if a changeset fails to create. Possibly recreate and re-run notifyhd
    }
+
+// function to close connection to TP and remove unwanted data in WDB and RDB's 
+eopdatacleanup:{[dict]
+    hclose each exec w from .servers.SERVERS where proctype = `segmentedtickerplant;
```

Comment:
I don't think we should hardcode the proctype - if you have a look at the subscription logic in e.g. rdb etc., I think there's a config variable called `tickerplanttypes` or something similar. It's possible that users could change the proctype etc.

I think we need to either (i) use the `tickerplanttypes` config variable or (ii) use the proctype(s)/handle(s) from the `.sub.SUBSCRIPTIONS` table

Reply:
Please see the latest approach in the recent commit, spoken to zan and she has suggested we pull the handle itself to from `.sub.SUBSCIPTIONS`